src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -364,6 +386,94 @@ def load_cgf_exposure(key: str, location: str) -> pd.DataFrame:
     return data
 
 
+def load_gbd_2021_exposure(key: str, location: str) -> pd.DataFrame:
```

Comment:
this function is slightly different from the one in ciff_sam, which was not returning reasonable results (based on looking at ciff sam saved artifact). This seems to work, but if there are V&V problems, this might be a good first suspect.